### ISSUE TITLE
[1.11.2] Builder/Quarry causes server crash : Ticking Block Entity

### ISSUE THREAD

**Issue by Cntrldfusion** [McJty/RFTools#1304]
MC Vers 1.11.2. Forge 13.20.1.2393
RFTools 1.1x-6.12
mcjtylib. 1.1x-2.4.3

Running a builder with a clearing quarry shape card.  Powered by a power cell.  Output to modular storage.  Seems to cause the server to crash every time it gets down close to bedrock.  This was reported by one of my players so I tried to reproduce it.  I could not reproduce it on single player mode.  However I was able to reproduce it on the server.  And it seems like it will continue to crash until it gets past whatever is causing it until it does the next chunk and hits it again.  I noticed crashes around the y level 8-12 area each time which is why my best guess is something to do with lava.

I am attaching several crash logs for you to look through.

[crash-2017-07-23_20.21.30-server.txt](https://github.com/McJty/RFTools/files/1168765/crash-2017-07-23_20.21.30-server.txt)
[crash-2017-07-23_20.23.23-server.txt](https://github.com/McJty/RFTools/files/1168768/crash-2017-07-23_20.23.23-server.txt)
[crash-2017-07-23_20.26.32-server.txt](https://github.com/McJty/RFTools/files/1168770/crash-2017-07-23_20.26.32-server.txt)
[crash-2017-07-23_23.19.58-server.txt](https://github.com/McJty/RFTools/files/1168766/crash-2017-07-23_23.19.58-server.txt)
[crash-2017-07-23_23.22.21-server.txt](https://github.com/McJty/RFTools/files/1168771/crash-2017-07-23_23.22.21-server.txt)
[crash-2017-07-23_23.29.21-server.txt](https://github.com/McJty/RFTools/files/1168767/crash-2017-07-23_23.29.21-server.txt)
[crash-2017-07-23_23.33.00-server.txt](https://github.com/McJty/RFTools/files/1168769/crash-2017-07-23_23.33.00-server.txt)

**Comment by McJty** [comment]
It might be a bug in another mod because it seems to be crashing in a drop handler that I don't recognize but I am on vacation right now so will have to look later

**Comment by Cntrldfusion** [comment]
If your talking about this line   "nc.handler.DropHandler.addBlockDrop", that could be Nuclearcraft although the quarry seems to mine the NC ores just fine.  Have a good vacation.  Will wait till you get back and confirm.

**Comment by Cntrldfusion** [comment]
Turns out it was a Nuclearcraft issue.  Something to do with checking for silk touch.  The drop handler was for rhodochrosite to drop when you mine red stone ore.  After I downloaded their patched mod file the quarry ran fine.  I'll put a link here just in case someone comes along with this same problem before those guys make it part of their next official release.

https://github.com/turbodiesel4598/NuclearCraft/issues/124
